src/Reporter.js

```diff
@@ -1,24 +1,30 @@
 // Internal: CavyReporter is responsible for sending the test results to
 // the CLI.
 export default class CavyReporter {
+  constructor() {
+    // Set the Reporter type to inProgress.
+    this.type = 'inProgress';
```

Comment:
Tested locally now and looks grand - this is still my only coment

Reply:
Hmm not sure I like either of those. How about `realtime` vs. `batch`? Or something similar?

Reply:
yeah i didn't when i wrote them haha.. `realtime` was literally the word i was trying to think of when i was stuck on `asynchronous`! so happy to go with that. `batch` sounds like you can have multiple batches though..

Reply:
@JonathanAndrews - what do you think about @jalada 's reporter name suggestions?

Reply:
@AbigailMcP @jalada 
`realtime` is nice 👍 
For the `simple`/`batch`, how about `bundle`?

Reply:
Hmm `bundle` makes me giggle for some reason... maybe because sounds like `bungle`. I think I prefer `batch`...

My only other suggestion is `deferred` - `realtime` vs `deferred`.

Reply:
I think so 👍